include/deal.II/lac/la_parallel_vector.h

```diff
@@ -614,6 +614,20 @@ namespace LinearAlgebra
       void
       copy_locally_owned_data_from(const Vector<Number2, MemorySpace> &src);
 
+      /**
+       * Import all the elements present in the distributed vector @p src.
+       * VectorOperation::values @p operation is used to decide if the elements
+       * in @p V should be added to the current vector or replace the current
+       * elements.
```

Comment:
Maybe you can explicitly add something like "The main purpose of this function is to get data from one memory space, e.g. CUDA, to the other, e.g. the host." It is clear from the template arguments and implicitly by the description above, but I think that being even more clear on this topic is never wrong.